arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java

```diff
@@ -487,6 +458,44 @@ public void populateColumnsForPrecompile(MessageFrame frame) {
     }
   }
 
+  // Support method for MODEXP
+  public static BigInteger computeExponentLog(
```

Comment:
It's surprising that this should return a `BigInteger`. As if any of `bbs` / `ebs` / `mbs` is large we `throw` anyway, and the end result is going to be at most `8 * 512`. Can't we use some `log2` method for (unsigned) `long`s or so ?